src/components/Images.vue

```diff
@@ -240,7 +240,7 @@ export default {
 				const growY = this.zoomHeight - this.height
 
 				this.shiftX = Math.min(Math.max(moveX, -growX / 2), growX / 2)
-				this.shiftY = Math.min(Math.max(moveY, -growY / 2), growX / 2)
```

Comment:
wow, that looks like a nice typo! Can't believe that hasn't been caught in all those years

Reply:
The key issue was incorrect calculation of top/left margins. I must admit that I am not fluent enough in CSS to fully understand why the shift must be multiplied by 2 to get the margins correct. But observations of the calculated box model in dev tools and testing confirmed this is the case. :innocent: